purchase_batch_invoicing/wizards/purchase_batch_invoicing.py

```diff
@@ -0,0 +1,118 @@
+# -*- coding: utf-8 -*-
+# Copyright 2016 Jairo Llopis <jairo.llopis@tecnativa.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+from logging import getLogger
+
+from openerp import _, api, fields, models
+from openerp.exceptions import UserError
+
+_logger = getLogger(__name__)
+
+
+class PurchaseBatchInvoicing(models.TransientModel):
+    _name = "purchase.batch_invoicing"
+
+    purchase_order_ids = fields.Many2many(
+        comodel_name="purchase.order",
+        string="Purchase orders",
+        domain="[('invoice_status', '=', 'to invoice')]",
```

Comment:
@pedrobaeza, [the field you ask already exists](https://github.com/odoo/odoo/blob/9.0/addons/purchase/purchase.py#L130), but you don't need it because this domain is enforced. All POs you see are ready to invoice.

The button suggestion has landed already :wink: